### DISCUSSION TITLE
Connecting first time error message

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#69] [General]
Trying to connect for very first time. The following error message appears. Hard refresh of browser and when tried a second time, the wallet connects successfully!

![Screenshot 2023-05-24 at 18 36 10](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/29dc935e-1646-4e2a-8714-3c945aeaedb8)
![Screenshot 2023-05-24 at 20 45 09](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/7360b8af-ba8e-4712-ac8c-b894b44716b8)

**Comment by TJ-BitCanna** [comment]
Are you able to reproduce? And include a screenshot of the console of your browser?

I'm not getting that error message, no matter what I try (clearing cache, removing chain and access from Keplr)

**Comment by dbwebapp** [comment]
see below : 
![Screenshot 2023-05-25 at 11 05 15](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/6a466f6e-8b9a-4094-a5e6-60fec2b91fa0)

**Comment by TJ-BitCanna** [comment]
Interesting, we have to dive into this.

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- debug

**Comment by TJ-BitCanna** [comment]
@artontray Can you retry and see what kind of error message Brave is giving you? It might be the AdBlocker is causing this issue.
![image](https://github.com/BitCannaGlobal/WeedWallet/assets/112879218/e11d0031-4787-4cf1-9418-de27a265c79e)

**Comment by dbwebapp** [comment]
Tried with Shield Off, could not reproduce the bug, so you may be right. AdBlocker could be the problem.
![Screenshot 2023-06-03 at 02 56 39](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/5fb49388-38b9-4577-adf5-367c0853bd92)

**Comment by TJ-BitCanna** [comment]
Ok for you to close this issue? @artontray

**Reply by dbwebapp** [reply]
Hello, 
I created a new session BRAVE with a new wallet KEPLR fresh and this problem come again when first time connected even with the adBlocker OFF . see the following : 

![Screenshot 2023-06-06 at 10 39 43](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/1549ed7a-dae1-44bb-8edf-42f1429a8f6d)
![Screenshot 2023-06-06 at 10 39 30](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/1933fd28-fc01-496a-9fc5-e1d1eed71b76)

**Comment by TJ-BitCanna** [comment]
Thanks! We'll dive into it after the first round of fixes being done as we speak :)

**Comment by TJ-BitCanna** [comment]
@artontray does this still occur?
